streams/src/main/java/org/apache/kafka/streams/processor/internals/ClientUtils.java

```diff
@@ -105,6 +109,7 @@ public static String getTaskProducerClientId(final String threadClientId, final
                 endOffsets = future.get(timeout.toMillis(), TimeUnit.MILLISECONDS);
             }
         } catch (final TimeoutException | RuntimeException | InterruptedException | ExecutionException e) {
+            LOG.warn("listOffsets request failed due to ", e);
```

Comment:
```suggestion
            LOG.warn("listOffsets request failed.", e);
```

Thanks! (minor suggestion to make the log message more typical)